doc/examples/gestures/lib/main_overlapping_tapables.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:math' as math;
+
+import 'package:flame/extensions/vector2.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/material.dart';
+import 'package:flame/game.dart';
+import 'package:flame/components/position_component.dart';
+import 'package:flame/components/mixins/tapable.dart';
+
+void main() {
+  final game = MyGame();
+
+  final widget = Container(
+    padding: const EdgeInsets.all(50),
+    color: const Color(0xFFA9A9A9),
+    child: game.widget,
+  );
+
+  runApp(widget);
+}
+
+class TapableSquare extends PositionComponent with Tapable {
+  Paint _randomPaint() {
+    final rng = math.Random();
+    final color = Color.fromRGBO(
+        rng.nextInt(256), rng.nextInt(256), rng.nextInt(256), 0.9);
+    return PaletteEntry(color).paint;
+  }
+
+  Paint currentPaint;
+
+  TapableSquare({Vector2 position}) {
+    currentPaint = _randomPaint();
+    size = Vector2.all(100);
+    this.position = position ?? Vector2.all(100);
+  }
+
+  @override
+  void render(Canvas canvas) {
+    super.render(canvas);
+    canvas.drawRect(size.toRect(), currentPaint);
+  }
+
+  @override
+  bool onTapUp(TapUpDetails details) {
+    return false;
```

Comment:
so one way to avoid the boolean is do what JS does, change the param for an Event<T> that has the event (TapUpDetails in this case) + a stopPropagation method. but I think the boolean is nice enough.